Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "serwus"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
imo it should be minor bump - features were added like `SanitizedString`

Reply:
Let's not be so greedy about versions bumping before 1.0 ;-)